blog/2019/08/release-process.md

```diff
@@ -37,7 +37,8 @@ This information is collected from a small set of posts on [discourse](https://d
 
 - Minor releases are also where significant refactorings of internals go, since we should only be refactoring to the extent that is necessary for fixing bugs in patch releases. This means that if you're relying on some internal Julia stuff that's not public, your code might break in a minor release. This is allowed according to SemVer since the change isn't to a public API—so technically it can break at any time; but we will avoid this in patch releases, so minor releases will be where you have to watch out if you rely on internals somehow.
 
-- Minor releases are branched every **four months**, which means that there are three minor releases per year. The rate is controlled by doing timed features freezes for minor releases: every four months, we announce on discourse that the current development version is about to feature freeze (with about two weeks notice); then on the freeze date, we branch a `release-1.3` branch for the minor release and after that no more feature development is allowed on that branch, from which the release will be tagged. More on that process below.
+- ~~Minor releases are branched every **four months**, which means that there are three minor releases per year. The rate is controlled by doing timed features freezes for minor releases: every four months, we announce on discourse that the current development version is about to feature freeze (with about two weeks notice); then on the freeze date, we branch a `release-1.3` branch for the minor release and after that no more feature development is allowed on that branch, from which the release will be tagged. More on that process below.~~
```

Comment:
Is it necessary to keep the old text that was probably never applied anyway?

Reply:
My thinking was that because this is a blogpost, it would be weird to delete content. So my original plan was to strikethrough the original content.

Now, I'm actually wondering if we should just preserve the original blogpost as-is, and just make a new page (either in the Julia devdocs, or somewhere in the Julia website) with an updated version of this content. And then, at the top of this blog post, just add a note that says:

> This is a historical blog post that is now outdated. The updated version of this content is maintained here: [link]

Reply:
I think that would be better.